spec/models/user_spec.rb

```diff
@@ -3,6 +3,23 @@
 require "rails_helper"
 
 RSpec.describe User, type: :model do
+  describe "password" do
+    context "password 8 characters long" do
+      it "user valid" do
+        user = build(:user, password: "12345678")
+        expect(user).to be_valid
+      end
+    end
+
+    context "password less than 8 characters long" do
+      it "user not valid" do
+        user = build(:user, password: "1234")
```

Comment:
This test would still pass even without the devise config change.

Reply:
Always have a failing test before making a change.

Reply:
Agree, but why would this pass without the config change? It does mention 8 chars in the expectation.

Reply:
Before we had 6 characters, so in order to fail before the change, the spec would need to be 6 or 7 characters long.